library/src/main/java/com/codetroopers/betterpickers/calendardatepicker/CalendarDatePickerDialogFragment.java

```diff
@@ -144,22 +144,7 @@
 
     public CalendarDatePickerDialogFragment() {
         // Empty constructor required for dialog fragment.
-    }
-
-    public boolean isThemeDark() {
-        return mStyleResId == R.style.BetterPickersRadialTimePickerDialog_Dark;
-    }
-
-    public void setThemeDark(boolean dark) {
-        if (dark) {
-            mStyleResId = R.style.BetterPickersRadialTimePickerDialog_Dark;
-        } else {
-            mStyleResId = R.style.BetterPickersRadialTimePickerDialog;
-        }
-    }
-
-    public void setThemeCustom(int styleResId) {
-        this.mStyleResId = styleResId;
+        mStyleResId = R.style.BetterPickersRadialTimePickerDialog;
```

Comment:
This looks like a no-setter : we always set the value taken from the theme... Am I missing something ?